Commands/notify.py

```diff
@@ -0,0 +1,267 @@
+        list_name = list_name.lower()
+
+        guild_data = await get_guild_data(ctx.message.guild.id)
+        msg_string = guild_data.notify(list_name)
+
+        await ctx.send(msg_string)
+
+    async def wait_for_added_reactions(self, ctx, msg, guild_data, timeout):
+        while True:
+            try:
+                reaction, user = await ctx.bot.wait_for(
+                    "reaction_add",
+                    check=lambda reaction, user: reaction.message.id == msg.id
+                    and not user.bot,
+                    timeout=30.0,
+                )
+
+                if reaction.custom_emoji:
+                    reaction_emoji = str(reaction.emoji.id)
+                else:
+                    reaction_emoji = reaction.emoji
+
+                for key, v in guild_data.notification_lists.items():
+
+                    if reaction_emoji == v["emoji"]:
+
+                        msg_string = await guild_data.sub_user(key, user.id)
+                        await ctx.send(msg_string)
+
+            except asyncio.TimeoutError:
+                pass
+
+            if time.time() > timeout:
+                break
+
+    async def wait_for_removed_reactions(self, ctx, msg, guild_data, timeout):
+        while True:
+            try:
+                reaction, user = await ctx.bot.wait_for(
+                    "reaction_remove",
+                    check=lambda reaction, user: reaction.message.id == msg.id
+                    and not user.bot,
+                    timeout=30.0,
+                )
+                if reaction.custom_emoji:
+                    reaction_emoji = str(reaction.emoji.id)
+                else:
+                    reaction_emoji = reaction.emoji
+                for key, v in guild_data.notification_lists.items():
+
+                    if reaction_emoji == v["emoji"]:
+
+                        msg_string = await guild_data.unsub_user(key, user.id)
+                        await ctx.send(msg_string)
+
+            except asyncio.TimeoutError:
+                pass
+
+            if time.time() > timeout:
+                break
+
+    @commands.command(name="show_lists")
+    async def show_lists(self, ctx):
+        # TODO printing of custom emojis
+        guild_data = await get_guild_data(ctx.message.guild.id)
+
+        if guild_data.notification_lists:
+            text = "Lists:\n"
+            for k, v in guild_data.notification_lists.items():
+                if v["is_custom_emoji"]:
+                    # TODO: Extract to 'get_custom_emoji' method for reusability
+                    text += ("\n<:"
+                             + ctx.bot.get_emoji(int(v["emoji"])).name
+                             + ":"
+                             + v["emoji"]
+                             + "> - "
+                             + k
+                             )
+                else:
+                    text += "\n" + v["emoji"] + " - " + k
+            # text += "\n".join(
```

Comment:
Is there a reason this is commented out? Can't we just remove it?

Reply:
This can be removed.